### ISSUE TITLE
Some files could not be processed

### ISSUE THREAD

**Issue by carrieyummy** [adamgruber/mochawesome-report-generator#125]
win10, mocha versions:
Line 260: +-- mocha@5.2.0
	Line 277: +-- mochawesome@3.1.1
	Line 285: | +-- mochawesome-report-generator@3.1.5 deduped
	Line 289: +-- mochawesome-merge@1.0.7
	Line 347: `-- mochawesome-report-generator@3.1.5
command:
npx mochawesome-report-generator mochawesome.json
![image](https://user-images.githubusercontent.com/39080899/54325328-99926100-463c-11e9-9489-48c523a61df5.png)
 
attached the json file.
[mochawesome.json.zip](https://github.com/adamgruber/mochawesome-report-generator/files/2964338/mochawesome.json.zip)

**Comment by adamgruber** [comment]
I was able to copy the contents of your JSON file into a new file and then it worked. Not sure what the issue is.

**Comment by tijnster** [comment]
I am having the same issue.. Locally I get the same error as described above, when running from CI it generates an empty html report..

**Comment by acenturion** [comment]
i have the same issue, the problem is the encoding file.

i change the coding file from UCS-2 BE BOM to UTF-8 with notepad++ in "encoding option"

and try again

**Comment by MehranShafqat** [comment]
yeah having same issue :( using v 3.1.5 and even tested with latest version still getting same issue

But after changing the encoding to UTF-8 its working.
@acenturion  now question is how we can change this encoding automatically ?

**Comment by AtulKrSharma** [comment]
[adamgruber](https://github.com/adamgruber)  I'm facing the below issue:

![image](https://user-images.githubusercontent.com/49794348/214658319-7a97c573-3247-4faf-847d-269fe237532c.png)

Name and Versions used in package.json:- 
    "mochawesome": "^7.1.3",
    "mochawesome-merge": "^4.2.2",
    "mochawesome-report-generator": "^6.2.0"

Configuration in cypress.config.js:-
module.exports = defineConfig({
  reporter: "mochawesome",
  reporterOptions: {
    reportDir: "cypress/results",
    overwrite: false,
    html: false,
    json: true,
  },

Commands used in sequence:

1) 
npm install --save-dev mochawesome mochawesome-merge mochawesome-report-generator

2) 
cypress run --reporter mochawesome \
  --reporter-options reportDir="cypress/results",overwrite=false,html=false,json=true

3)
npx mochawesome-merge "cypress/results/*.json" > mochawesome.json

4)
npx marge mochawesome.json

Please note that I've tried the workaround too for taking "mochawesome.json" and converting it into UTF- 8 but to no relief
Please suggest.
